src/main/java/com/google/api/tools/framework/aspects/versioning/model/VersionAttribute.java

```diff
@@ -16,14 +16,14 @@
 
 package com.google.api.tools.framework.aspects.versioning.model;
 
+import com.google.api.tools.framework.model.Interface;
```

Comment:
So, it javadoc doesn't follow the rules for finding java classes (e.g. the fully qualified package.class), but requires that you have java "import" statements to satisfy itself?  Neat...

I'm assuming linter type tools will recognized that import as being "used"?